socialhome/content/models.py

```diff
@@ -114,10 +115,17 @@ def save(self, author=None, *args, **kwargs):
         if self.pk:
             # Old instance, bust the cache
             self.bust_cache()
-        elif author:
-            # New with author, set a GUID and author
-            self.guid = uuid4()
-            self.author = author
+        else:
+            if author:
+                # New with author, set a GUID and author
+                self.guid = uuid4()
+                self.author = author
+
+            if self.pinned:
+                max_order = Content.objects.values_list("order").distinct().aggregate(order=Max("order"))["order"]
```

Comment:
> Way shorter and a lot more performant is:
> ```
> Content.objects.aggregate(Max("order"))["order__max"]
> ```

I believe you but I don't understand. I remember during my studies my teachers telling me that filtering content like using `DISTINCT` improved queries performance.